cms/urls.py

```diff
@@ -224,3 +224,7 @@
     url(r'^404$', handler404),
     url(r'^500$', handler500),
 )
+
+urlpatterns += (
```

Comment:
Actually, since we're not allowing sneakpeek access to CMS, we can remove this `urlpattern`.